File: Assignment42/program1.cpp
/////////////////////////////////////////////////////////////////// 
//
// File name : program1.cpp
// Description : Pattern printing using recursion
// Author : Adinath Santosh Pawar
// Date : 21/7/25
// 
///////////////////////////////////////////////////////////////////

#include<iostream>
using namespace std;

void Display(int iNo)
{
    static int i = iNo;

    if(i > 0)
    {
        cout<<i<<" * ";
        i--;
        Display(i);
    }
}


int main()
{
    Display(5);

    return 0;
}

File: Assignment48/program1.java
import java.util.*;

/////////////////////////////////////////////////////////////////// 
//
// File name : program1.java
// Description : Pattern printing
// Author : Adinath Santosh Pawar
// Date : 20/8/25           
// 
///////////////////////////////////////////////////////////////////

class Pattern
{
    public void Pattern(String str)
    {
        for(int i = 0; i < str.length(); i++)
        {
            System.out.print(str.charAt(i)+" ");
        }
    }
}

class program1
{
    public static void main(String A[])
    {
       Pattern pobj = new Pattern();
       Scanner sobj = new Scanner(System.in);

       System.out.print("Enter the string ");
       String str = sobj.nextLine();

       pobj.Pattern(str);
    }
}